### DISCUSSION TITLE
Issue with IP Pool Feature: All Emails Being Sent from Primary IP Despite Configurations

### DISCUSSION THREAD

**Discussion by ehsandiary** [postalserver/postal#3067] [Help with using Postal] [answered]
Hello everyone,

I have a question regarding the IP pool settings.

I have correctly set the network card settings on my machine, and when I test using curl from each of my IPs, the requests go out and return to the server using the same IP. I have also correctly set the IP pool settings in the Postal configuration file and its dashboard, setting the priority for the second IP to 100 and the main IP to 1. However, all my emails are still being sent from the first IP. 

How can I check and troubleshoot this?
![image](https://github.com/user-attachments/assets/dc293360-887c-4d5d-8c9f-c1272e04c47f)
![image](https://github.com/user-attachments/assets/1feea929-f68b-46c8-b8ec-0a3718700302)

**Comment by TR-SametOzen** [comment]
Hello,

I have the same problem. Did you solve it?

**Comment by bukunmilab** [comment]
Still looking for an answer too.

**Comment by z3d3m0n** [comment]
i have the same issue , have you found a solution?

**Comment by ehsandiary** [comment, marked as answer]
Yes, we found a solution for our case:

Since we haven't created any rules for the IP pool yet, the IP pool has simply been ignored. By creating a rule, we could now successfully send through our second IP

To use the IP Pool feature, it is necessary to set the rule for it

**Reply by bukunmilab** [reply]
Can you share this rule and how we can implement it?

**Reply by z3d3m0n** [reply]
yes i have found an option too
in this thread
https://github.com/orgs/postalserver/discussions/2752

but is it a bug that ip pool only works when the ip rules are set?

**Reply by ehsandiary** [reply]
> yes i have found an option too in this thread https://github.com/orgs/postalserver/discussions/2752
> 
> but is it a bug that ip pool only works when the ip rules are set?

Totally agree with you, they didn't mention that in the documents 😅

**Reply by ehsandiary** [reply]
> Can you share this rule and how we can implement it?

Sure 😊
As you can see in the picture, in the rules section, define the domains you want to send with for the desired IP Pool.

![image](https://github.com/user-attachments/assets/29c2e9af-40e8-4874-9e98-b372306e2f47)
